application/util/uri-helper.js

```diff
@@ -28,9 +30,25 @@ module.exports = {
      * @return {String}        URI with replaced parameter
      */
     injectValueIntoUri : function(name, uri, value) {
-        var regex = new RegExp(':' + name + '(?![A-z])');
+        console.log(value);
+        // inject array differently using name[]=value
+        if (typeof value ==='object') {
+            // skip if someone tries to put an object in a uri
+            if (_.isArray(value)) {
```

Comment:
Would this expression ever resolve to `true`? We're only running this code if `typeof value ==='object'`.